### PR TITLE
Generate net/fdb boilerplate.

### LINKED ISSUE #239: Implement net/fdb net/tunnels net/tunnel net/vlans

### PR DESCRIPTION
**Pull request by zancas**
@swormke
Issues:
Fixes #239

Problem: lbaasv2 requires these objects

Analysis: Significant boilerplate has been autogenerated.

Tests: No new tests.

### CODE REVIEW

**Comment by swormke** [comment]
Can you please add docstrings like the ones that we already have?  You can look at something like VLAN to see the format.  I would like to see them consistent for the v0.1.1 release in the docs.

**Comment by swormke** [comment]
Also you need to add the .rst entries for these as well.

**Comment by zancas** [comment]
@pjbreaux Here you are sir.

File: utils/template_engine.py [hunk 1]
```diff
@@ -149,7 +162,10 @@ def _format_resource(self, kind, raw_conf):
                                       kind=kind,
                                       attr_reg_dict={})
         import_str = 'from f5.bigip.resource import Resource'
-        python_as_string = import_str + 3*os.linesep + Resourcestr + os.linesep
+        mdoc_string = self._build_module_docstring(kind, raw_conf)
```
Comment (pjbreaux) [inline utils/template_engine.py]: Think about functionalizing this operation.

**Comment by pjbreaux** [comment]
Ship it!

**Pull request merged**